### DISCUSSION TITLE
搞得跟洗脑一样的

### DISCUSSION THREAD

**Discussion by ldxhhs** [The-Run-Philosophy-Organization/run#659] [General]
根据自己需求，想走就走，想留就留。每个人都经历都不一样。想要提升自己，就出去锻炼锻炼，如果是别的原因也无可厚非。

**Comment by mocuishle1998** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by jundaodao** [comment]
您的邮件我已经收到了，我会尽快回复您！                            ————焦江昆
